src/infrastructure/startup/GlobalServices.ts

```diff
@@ -3,6 +3,7 @@ import glob from "glob";
 import path from "path";
 import ISetup from "infrastructure/ISetup";
 
+// TODO: Add @Global decorator
```

Comment:
Instead of this, add to the global scope after importing. Also, generate the .d.ts for the global types.